bindata/network/additional-networks/simplemacvlan/simplemacvlan.yaml

```diff
@@ -0,0 +1,19 @@
+---
+apiVersion: "k8s.cni.cncf.io/v1"
+kind: NetworkAttachmentDefinition
+metadata:
+  name: {{.AdditionalNetworkName}}
+  namespace: default
+spec:
+  config: '{
+    "cniVersion": "0.3.0",
```

Comment:
This should be v0.3.1, but they're actually identical... long story. This can be fixed later.